com.musala.xml.parser/src/com/musala/xml/parser/Enums.java

```diff
@@ -0,0 +1,4 @@
+package com.musala.xml.parser;
+
+
+enum  ClassNumberText{first, second, third, fourth, fifth, sixth, seventh, eighth, ninth, nth}
```

Comment:
Since you have got only one enum named ClassNumberText, the Enums.java file should be renamed to ClassNumberText.java.